[Email Security] Add TTL note (#22465)
cloudflare/cloudflare-docs: src/content/partials/cloudflare-one/email-security/deployment/mx-deployment-prerequisites.mdx

Changing the TTL value instructs DNS servers on how long to cache this value before requesting an update from the responsible nameserver. You need to change the TTL value before changing your MX records to Email Security. This will ensure that changes take effect quickly and can also be reverted quickly if needed. If your DNS manager does not allow for a TTL of five minutes, set it to the lowest possible setting.

:::note

> TTL changes should be done a few days before the production update and wait at least the time of the old values to expire before making the production changes, as senders may still be caching the old value with the longer TTLs. 

Reviewer: ```suggestion
Make TTL changes a few days before the production update, and wait at least as long as the old TTL values before making the update, since some senders might still be using the old cached values.
```
